augur/api/view/utils.py

```diff
@@ -209,67 +126,6 @@ def toCacheFilepath(endpoint, append = True):
 def toCacheURL(endpoint):
     return getSetting('approot') + str(toCacheFilepath(endpoint))
 
-""" ----------------------------------------------------------------
-requestJson:
-    Attempts to load JSON data from cache for the given endpoint.
-    If no cache file is found, a request is made to the URL for
-    the given endpoint and, if successful, the resulting JSON is
-    cached for future use. Cached files will be stored with all
-    '/' characters replaced with '.' for filesystem compatibility.
-
-@PARAM:     endpoint: String
-        A String representation of the requested
-        json endpoint (relative to the api root).
-
-@RETURN:    data: JSON
-        An object representing the JSON data read
-        from either the cache file or the enpoint
-        URL. Will return None if an error isreturn None
-        encountered.
-"""
-def requestJson(endpoint, cached = True):
-    filename = toCacheFilepath(endpoint)
-    requestURL = getSetting('serving') + "/" + endpoint
-    logger.info(f'requesting json from: {endpoint}')
-    try:
-        if cached and cacheFileExists(filename):
-            with open(filename) as f:
-                data = json.load(f)
-        else:
-            with urllib.request.urlopen(requestURL) as url:
-                if url.getcode() != 200:
-                    raise urllib.error.HTTPError(code = url.getcode())
-
-                data = json.loads(url.read().decode())
-
-                if cached:
-                    with open(filename, 'w') as f:
-                        json.dump(data, f)
-        if filename in cache_files_requested:
-            cache_files_requested.remove(filename)
-        return data
-    except Exception as err:
-        logger.error("An exception occurred while fulfilling a json request")
-        logger.error(err)
-        return False, str(err)
-
-""" ----------------------------------------------------------------
-"""
-def requestPNG(endpoint):
```

Comment:
Don't the Visualization endpoints rely on this method?

Reply:
No, the visualization endpoints do not rely on any methods in the fontend code.

If you are talking about the report page which uses the visualization endpoints, those use a different function for downloading and caching the images.